### PR TITLE
Add a `DynamoSyncInputExecuteTime` counter

### PR DESCRIPTION
**Pull request by JackCaoG**
Aot-autograd might apply reshape on the input before passing to `openxla` backend, in that case we need to execute to sync the input. Add a counter to track that.

### CODE REVIEW

File: test/dynamo/test_dynamo.py [hunk 1]
```diff
@@ -124,6 +124,7 @@ def test_simple_model(self):
     xla_y3 = xla_y * 3
     res_xla_dynamo_3 = fn_simple_dynamo(xla_xy, xla_y3)
     res_cpu_3 = self.fn_simple(x + y, y * 3)
+    breakpoint()
```
Comment (wonjoolee95) [inline test/dynamo/test_dynamo.py:127]: nit: remove 😄

**Comment by wonjoolee95** [comment]
LGTM, the `breakpoint` seems to be causing the CI to fail, but otherwise LGTM.

File: test/dynamo/test_dynamo.py (hunk 1 above)
Comment (JackCaoG) [reply test/dynamo/test_dynamo.py:127]: lol

**Comment by JackCaoG** [comment]
Let me rebase this pr

**Comment by JackCaoG** [comment]
opps, @wonjoolee95 do you mind stamping this pr?

**Pull request merged**